yt_dlp/extractor/mlb.py

```diff
@@ -288,69 +289,172 @@ class MLBTVIE(InfoExtractor):
             'skip_download': True,
         },
     }]
-    _access_token = None
+    _GRAPHQL_INIT_QUERY = '''mutation initSession($device: InitSessionInput!, $clientType: ClientType!, $experience: ExperienceTypeInput) {
```

Comment:
i think this is easier to read with 
```suggestion
    _GRAPHQL_INIT_QUERY = '''\
mutation initSession($device: InitSessionInput!, $clientType: ClientType!, $experience: ExperienceTypeInput) {
```